internal/v2/resources/apono_access_flow_v2_resource_unit_test.go

```diff
@@ -0,0 +1,308 @@
+package resources
+
+import (
+	"context"
+	"testing"
+
+	"github.com/apono-io/terraform-provider-apono/internal/v2/api/mocks"
+	"github.com/apono-io/terraform-provider-apono/internal/v2/models"
+	"github.com/apono-io/terraform-provider-apono/internal/v2/testcommon"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-go/tftypes"
+	"github.com/ogen-go/ogen/validate"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAponoAccessFlowV2Resource(t *testing.T) {
```

Comment:
the unit tests are much cleaner now.
I took this approach because they were extremely large (especially this one).
this uses a the HTTP response and converts it to a model. Once we have the model and the response we can work with it and focus on the test, instead of defining crazy structs.